README.md

```diff
@@ -58,3 +58,7 @@ juju run-action <unit> remove-default-gw --wait
 
 - **lte-core**: The LTE core interface is used to connect to a 4G/LTE core network via its MME IPv4 address.
 
+### Sample charms implementing the interface
```

Comment:
I don't think we need this here, if it's anywhere it should be in the interface project

Reply:
Oh, for sure. Removed.